src/uwtools/cli.py

```diff
@@ -874,6 +884,7 @@ def _add_subparser_for_driver_task(
     _add_arg_batch(optional)
     _add_arg_dry_run(optional)
     _add_arg_graph_file(optional)
+    _add_arg_key_path(optional)
```

Comment:
This is where simple built-in integration tests would come in handy: If you'd added an integration test that executed, say, `uw upp --config-file config.yaml --key-path a.b [other options]`, it would fail because, without this line, `--key-path` would not be recognized.

But then I have to ask: How did you successfully test this new feature from the CLI without this line? I don't think you could have. What works best for me is doing the code development and manually testing the feature -- which for a CLI-exposed feature like this one would necessitate a command-line test -- until it appears to behave as I expect, and _then_ write unit tests, and _then_ write integration tests (someday soon!), and _then_ write documentation.